activity/models.py

```diff
@@ -23,6 +24,7 @@ def get_poster_path(instance, filename):
     topics = models.ManyToManyField(Skill, related_name='ProjectTopics', blank=True)
     detail = RichTextField(verbose_name='Details', max_length=10000, null=True)
     links = models.ManyToManyField(Portal, related_name='ProjectLinks', through='ProjectLink')
+    album = models.ForeignKey(Album, on_delete=models.CASCADE, null=True, blank=True, to_field='title')
```

Comment:
```to_field``` attribute is not required, It should automatically point to the primary key (ID of the album). At the same time, we shouldn't be using publicly visible primary key like title either.